ttfunk.gemspec

```diff
@@ -5,7 +5,6 @@ Gem::Specification.new do |spec|
   spec.platform = Gem::Platform::RUBY
   spec.summary = 'TrueType Font Metrics Parser'
   spec.description = 'Font Metrics Parser for the Prawn PDF generator'
-  spec.licenses = %w[Nonstandard GPL-2.0 GPL-3.0]
```

Comment:
Could you please keep this and remove the other one?

Reply:
Sure, done.